.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: node_js
 node_js:
-  - "6"
   - "8"
   - "10"
```

Comment:
Can you add Node.js 12 while changing `.travis.yml`? Not a big deal, we can add it later in a new PR.

Reply:
@bajtos Yep I can submit another PR to add Node.js 12, I moved 6 in this PR since CI fails.

Running juggler v3 and v4 causes 20+ failures, I am not sure are they all broken by the upgrading of juggler, but this PR and https://github.com/strongloop/loopback-connector-mysql/pull/392 needs to fix them all, which is already a big amount of work, so I would like to add Node 12 in a separate PR :)

Reply:
Makes sense :+1: